make/clang.inc

```diff
@@ -29,6 +29,10 @@ ifneq (,$(shell clang --version | egrep  "clang version [6-9]\.[0-9]\.[0-9]"))
 CPLUS_FLAGS += -fopenmp-simd #supported at least since 6.0 version
 endif
 
+ifneq (,$(shell clang --version | egrep  "DPC++"))
```

Comment:
It's minor issue, but `+` in regex means _any_ symbol, if you need to check for `+` symbols use backslashes `egrep "DPC\+\+"`